src/components/KTable/KTable.vue

```diff
@@ -1028,6 +1200,25 @@ export const defaultSorter = (key: string, previousKey: string, sortOrder: strin
       padding: var(--kui-space-50, $kui-space-50) var(--kui-space-60, $kui-space-60);
       text-align: left;
 
+      &.resizable {
```

Comment:
suggestion: Is it possible to limit the maximum width of the column headers as well? 

A user should never need to expand the column to a width greater than the column cell with the largest amount of content.

I believe there's logic in the KTruncate component that might be able to be repurposed here in order to determine when all of the cell content is visible, and at that point, restrict increasing the table width beyond that value. Otherwise, the user can infinitely expand the column width.

Reply:
I'm not sure I understand what you're saying here. If we don't allow columns to expand beyond the largest cell's width doesn't that mean we can't guarantee the table width? (ie. I want the table to span the entire page's width and I'd like the columns to have relatively even spacing. This means sometimes I will want columns with minimal content to take up more space than they need.

To a degree this also comes down to the user's preference. As long as the table is still functional (you can continue to use the resize/other controls) shouldn't we allow them to expand the table as wide as they want?

Reply:
I'm not sure I understand why a user would need to be able to expand a table cell extremely beyond the width of the cell's content?

Reply:
Not extremely beyond, I'm just not sure how to limit it when you may have a table like this one where it would look strange with the content not evenly spaced because we would need to enforce a `max-width`.

![image](https://github.com/Kong/kongponents/assets/67973710/94146cb8-04a0-4377-975b-6da3728b2ef4)

None of these columns content extends to the current width of the columns, so this would make the table much smaller. Each of the columns would immediately shrink to fit the content.

Reply:
what about maxing out at `100vw` or something?